glslc/test/include.py

```diff
@@ -51,6 +51,20 @@ class VerifyIncludeNotFound(expect.ErrorMessage):
     ]
 
 @inside_glslc_testsuite('Include')
+class VerifyIncludeCannotOpen(expect.ErrorMessage):
+    """Tests #including an existing but not readable sibling file."""
+
+    environment = Directory('.', [
+        File('a.vert', 'content a\n#include "b"\n'),
+        File('b', 'content b\n', readable=False)])
+
+    glslc_args = ['-E', 'a.vert']
+    expected_error = [
+        "a.vert:2: error: '#include' : Cannot open or find include file.\n",
+        '1 error generated.\n'
```

Comment:
The error message here is exactly the same as the include file 'NotFound' case. The error message in  libshaderc_util::ReadFile() doesn't show up here. This is because libshaderc_util::FileFinder's method: FindReadableFilePath() checks the access bits before calling ReadFile() so the call of ReadFile() in FileIncluder will never have an unreadable path to read.